homeassistant/config.py

```diff
@@ -669,6 +674,39 @@ def async_process_component_config(hass, config, domain):
     return None
 
 
+async def async_check_ha_config_file_new(hass, config_path=None):
```

Comment:
Please don't call it "new", that name doesn't age…

Reply:
It will completely replace the function above without the "_new" once done